frontend/src/components/Scheduling/SmartSchedulingDashboard.jsx

```
@@ -0,0 +1,489 @@
+      if (appointmentsResponse.data.success) {
+        setAppointments(appointmentsResponse.data.data || []);
+      }
+
+      // Generate insights based on user behavior
+      generateInsights();
+    } catch (error) {
+      console.error("Failed to load dashboard data:", error);
+    } finally {
+      setLoading(false);
+    }
+  }, [user, generateInsights]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)